Automattic/jetpack
# Sync: Blacklist the sl-insta-media Post Type

#### Changes proposed in this Pull Request:
The `sl-insta-media` data do not need be stored by Sync; by blacklisting the post type, we can improve Sync efficiency on sites that make use of the post type (1203088870910919-as-1203072621759423).

The changes will affect the following:
* Sync

#### Other information:


#### Jetpack product discussion
* None

#### Does this pull request change what data or activity we track or use?
* No

#### Testing instructions:
1. Activate the "Jetpack Debug Tools" plugin.
2. Go to Jetpack Debug > REST API Tester.
3. Checkout the `trunk` branch.
4. Make a `GET` request to `jetpack/v4/sync/settings`.
5. Look at the array values for the `post_types_blacklist` key. You *should not* see `sl-insta-media` in the array values.
6. Checkout `add/sync-sl-insta-media-to-blacklist`, the branch for this PR.
7. Make a `GET` request to `jetpack/v4/sync/settings`.
8. Look at the array values for the `post_types_blacklist` key. You *should* see `sl-insta-media` in the array values.
